app/controllers/api/v2/versions_controller.rb

```diff
@@ -9,6 +9,14 @@ def show
       respond_to do |format|
         format.json { render json: version }
         format.yaml { render yaml: version }
+        format.sha256 do
+          hashes = @rubygem.version_manifest(version_params[:number], version_params[:platform]).checksums_file
+          if hashes
+            render plain: hashes
+          else
+            render plain: "SHA256 format unavailable for this version.", status: :gateway_timeout
```

Comment:
`status: :not_found` ?

Reply:
I was thinking of queuing the job here. The unavailability is temporary, more like we couldn't reach AWS and it will work in a moment.

I'm fine with not found too. I was going back and forth on this.